man/systemd.network.xml

```diff
@@ -840,6 +840,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+          <term><varname>IPv6ULASubnets=</varname></term>
```

Comment:
Need to provide a setting for the global id, as well as the subnets.

Reply:
No 3.2.2. describes how to do this.